src/main/groovy/org/labkey/gradle/plugin/FileModule.groovy

```diff
@@ -43,13 +43,13 @@ import java.util.regex.Matcher
  */
 class FileModule implements Plugin<Project>
 {
-    private static final Map<String, String> _foundModules = new HashMap<>();
-
     @Override
     void apply(Project project)
     {
         def moduleKey = project.getName().toLowerCase()
-        def otherPath = _foundModules.get(moduleKey)
+        ServerDeployExtension deployExt = BuildUtils.getServerProject(project).extensions.getByType(ServerDeployExtension.class)
```

Comment:
This will break standalone module builds (such as `tutorialModules/standalone`) since there isn't a `server` project.

Reply:
Ah, good catch. I'll likely just disable this if there isn't a server project, since this is just a nicety to fail early so other things don't go haywire, right?

Reply:
Correct.